py/pystack.h

```diff
@@ -26,6 +26,8 @@
 #ifndef MICROPY_INCLUDED_PY_PYSTACK_H
 #define MICROPY_INCLUDED_PY_PYSTACK_H
 
+#include <assert.h>
```

Comment:
Should maybe be within the `MICROPY_ENABLE_PYSTACK` ?

Reply:
Yes, good idea.